modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -533,7 +541,19 @@ class ApiClient(object):
                 if auth_setting['in'] == 'cookie':
                     headers['Cookie'] = auth_setting['value']
                 elif auth_setting['in'] == 'header':
-                    headers[auth_setting['key']] = auth_setting['value']
+                    if auth_setting['type'] != 'http-signature':
+                        headers[auth_setting['key']] = auth_setting['value']
+{{#hasHttpSignatureMethods}}
+                    else:
+                        # The HTTP signature scheme requires multiple HTTP headers
+                        # that are calculated dynamically.
+                        signing_info = self.configuration.signing_info
+                        # NOTE: no need to raise an exception because this entry only exists
```

Comment:
Can you delete this note line and the comment line below it?
They are not needed